src/Common/src/System/Resources/ResourceWriter.cs

```diff
@@ -2,17 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-/*============================================================
-**
-** Class:  ResourceWriter
-**
-**
-**
-** Purpose: Default way to write strings to a CLR resource
-** file.
-**
-**
-===========================================================*/
+#nullable enable
```

Comment:
Why is this needed?  Is this file included in another project that doesn't set `<Nullable>enable</Nullable>`?

Reply:
Yes, I think it was S.Resources.Extensions.